setup.py

```diff
@@ -167,18 +185,36 @@ def run_tests(self):
     package_data=PACKAGE_DATA,
     ext_modules=[
         Extension('jwst.tweakreg.chelp',
-            glob('src/tweakreg/*.c'),
-            include_dirs=[np_include()],
-            define_macros=[('NUMPY', '1')]),
+                  glob('src/tweakreg/*.c'),
+                  include_dirs=[np_include()],
+                  define_macros=[('NUMPY', '1')]),
     ],
     install_requires=[
-        'namedlist'
-    ],
-    tests_require=[
-        'pytest',
+        'asdf>=2.1',
+        'astropy>=3.0',
```

Comment:
Let's bump `astropy>=3.1`, as that has now been released (and is required) since you first started this PR.  We'll have to bump `asdf` too once the next release is made.